event-provider/actions/publish-events/index.js

```diff
@@ -0,0 +1,91 @@
+/*
+* <license header>
+*/
+
+/**
+ * This is a sample action showcasing how to create a cloud event and publish to I/O Events
+ *
+ * Note:
+ * You might want to disable authentication and authorization checks against Adobe Identity Management System for a generic action. In that case:
+ *   - Remove the require-adobe-auth annotation for this action in the manifest.yml of your application
+ *   - Remove the Authorization header from the array passed in checkMissingRequestInputs
+ *   - The two steps above imply that every client knowing the URL to this deployed action will be able to invoke it without any authentication and authorization checks against Adobe Identity Management System
+ *   - Make sure to validate these changes against your security requirements before deploying the action
+ */
+
+
+const { Core, Events } = require('@adobe/aio-sdk')
+const uuid = require('uuid')
+const cloudEventV1 = require('cloudevents-sdk/v1')
+const { errorResponse, getBearerToken, stringParameters, checkMissingRequestInputs } = require('../utils')
+const { context, getToken } = require('@adobe/aio-lib-ims')
```

Comment:
Same as above - do you have to use aio-lib-ims or is there another way to get the token ?